server/libs/domain/src/system-config/system-config.initial.ts

```diff
@@ -0,0 +1 @@
+export const INITIAL_SYSTEM_CONFIG = 'INITIAL_SYSTEM_CONFIG';
```

Comment:
What is your plan for this file, what are additional information we will be using in this file?

Reply:
Just needed to put it somewhere else to not have a circular dependency, I merged it into the datetime file and renamed to `*.constants.ts*`